OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -66,7 +66,7 @@ static string[] QueryDevices(string label, int type)
 			do
 			{
 				var str = Marshal.PtrToStringAuto(next);
-				next += UnicodeEncoding.Default.GetByteCount(str) + 1;
+				next += Encoding.Default.GetByteCount(str) + 1;
```

Comment:
I might cause problems on systems that don't really work with Unicode properly. Saw some gibberish when running bleed on Windows lately. This might use the system one, which is a good compromise from https://github.com/OpenRA/OpenRA/pull/11183/commits/f22bc68d948531b90a7e9ce9dc6f93e021007836. This still should count the bytes properly.